dev-util/shards/shards-0.15.0.ebuild

```diff
@@ -20,7 +20,7 @@ SLOT="0"
 KEYWORDS="~amd64"
 
 DEPEND="
-	>dev-lang/crystal-0.11.1[yaml]
+	>dev-lang/crystal-0.11.1[yaml(+)]
```

Comment:
revbump, ok;
I'm not sure adding one of crystal's dependencies to the shards ebuild is the best way forward. 
It wouldn't be very clear what that dependency is doing, since shards itself isn't directly using or linking to libyaml, only using it through crystal's interface.
This dependency should communicate "crystal with the ability to use the yaml feature"

Reply:
No matter how you look at it, the dep is then 
```
	dev-lang/crystal
	dev-libs/libyaml
```
and that's correct.